CHANGES/100.feature

```diff
@@ -0,0 +1 @@
+Added label commands.
```

Comment:
:thinking:

Reply:
:exploding_head:

Reply:
We don't check that a changelog is only added when there is also a `#fixes` stanza, right?

Reply:
Yep, the [wrong] assumption is that there wouldn't be a changelog if there is no `fixes #...`